### PR TITLE
Updated the properties table

### PR DESCRIPTION
**Pull request by KBbitsP**
Updated the properties table

### CODE REVIEW

File: userguide/platform/includes/killbill-configuration-properties-table.adoc [hunk 1]
```diff
@@ -510,7 +510,7 @@ KB_org_killbill_billing_server_event_post_type_skip   |List of event types to be
 KB_org_killbill_billing_server_event_dispatch_type_skip   |List of event types to be skipped (not dispatched internally)   |-   |Per-Tenant/Config File/Environment Variable
 |org.killbill.billing.server.event.bulk.subscription.aggregate/
 
-KB_org_killbill_billing_server_event_bulk_subscription_aggregate   |Coming Soon  |false   |Per-Tenant/Config File/Environment Variable
+KB_org_killbill_billing_server_event_bulk_subscription_aggregate   |Aggregate the events for bulk subscription  |false   |Per-Tenant/Config File/Environment Variable
```
Comment (reshmabidikar) [inline userguide/platform/includes/killbill-configuration-properties-table.adoc:513]: Maybe create a PR to also update it [here](https://github.com/killbill/killbill/blob/33aa3e9a65bf40de10edee7429341370488d4fb5/util/src/main/java/org/killbill/billing/util/config/definition/EventConfig.java#L53)?

File: userguide/platform/includes/killbill-configuration-properties-table.adoc [hunk 2]
```diff
@@ -976,7 +976,7 @@ KB_org_killbill_security_skipAuthForPlugins   |Specifies whether authentication
 |===
 |Property Name/Environment Variable Name   |Description   |Default Value | Configuration Method
 //-------------------------------------------------
-|org.killbill.catalog.frequentValuesCacheSize   |Coming Soon   |1000   |System property
+|org.killbill.catalog.frequentValuesCacheSize   |Number of values to be considered for cache   |1000   |System property
```
Comment (reshmabidikar) [inline userguide/platform/includes/killbill-configuration-properties-table.adoc]: I believe this is used in the [DefaultPrice](https://github.com/killbill/killbill/blob/33aa3e9a65bf40de10edee7429341370488d4fb5/catalog/src/main/java/org/killbill/billing/catalog/DefaultPrice.java#L44) class. So, maybe specify that it is applicable for price and elaborate it slightly? The logic for this is [here](https://github.com/killbill/killbill/blob/33aa3e9a65bf40de10edee7429341370488d4fb5/catalog/src/main/java/org/killbill/billing/catalog/DefaultPrice.java#L88).

File: userguide/platform/includes/kaui-configuration-properties-table.adoc [hunk 3]
```diff
@@ -48,17 +48,17 @@ KILLBILL_URL/KAUI_KILLBILL_URL   |Specifies the Kill Bill URL   |http://127.0.0.
...
 KILLBILL_DISABLE_SSL_VERIFICATION/
 
-KAUI_KILLBILL_DISABLE_SSL_VERIFICATION   |Coming Soon   |Coming Soon
+KAUI_KILLBILL_DISABLE_SSL_VERIFICATION   |Whether to ignore SSL certificates checks   |false
 |kaui.read_timeout/
 
 KILLBILL_READ_TIMEOUT/
 
-KAUI_KILLBILL_READ_TIMEOUT   |Coming Soon   |60000
+KAUI_KILLBILL_READ_TIMEOUT   |Specifies the read timeout   |60000
```
Comment (reshmabidikar) [inline userguide/platform/includes/kaui-configuration-properties-table.adoc]: Does the specify the time after which Kaui times out if it is unable to connect to KB? If so, could you elaborate this?

File: userguide/platform/includes/kaui-configuration-properties-table.adoc [hunk 4]
```diff
@@ -48,17 +48,17 @@ KILLBILL_URL/KAUI_KILLBILL_URL   |Specifies the Kill Bill URL   |http://127.0.0.
...
 |kaui.connection_timeout/
 
 KILLBILL_CONNECTION_TIMEOUT/
 
-KAUI_KILLBILL_CONNECTION_TIMEOUT   |Coming Soon   |60000
+KAUI_KILLBILL_CONNECTION_TIMEOUT   |Specifies the connection timeout   |60000
```
Comment (reshmabidikar) [inline userguide/platform/includes/kaui-configuration-properties-table.adoc]: Same comment as above, we need to confirm if this is the KB connection timeout and if so elaborate this description

File: userguide/platform/includes/kaui-configuration-properties-table.adoc [hunk 5]
```diff
@@ -69,16 +69,16 @@ KAUI_KILLBILL_CONNECTION_TIMEOUT   |Coming Soon   |60000
 //-------------------------------------------------
 |kaui.demo/
 
-KAUI_DEMO_MODE   |Coming Soon   |Coming Soon
+KAUI_DEMO_MODE   |Whether to run Kaui in demo mode   |false
 |kaui.plugins_whitelist/
 
-KAUI_PLUGINS_WHITELIST   |Coming Soon   |None
+KAUI_PLUGINS_WHITELIST   |Kaui plugins to be whitelisted   |None
```
Comment (reshmabidikar) [inline userguide/platform/includes/kaui-configuration-properties-table.adoc]: This is a bit unclear to me. Maybe check with Kyle and elaborate this a bit?

**Review by reshmabidikar — COMMENTED** [review]
Left a few minor comments.

File: userguide/platform/includes/killbill-configuration-properties-table.adoc (hunk 1 above)
Comment (KBbitsP) [reply userguide/platform/includes/killbill-configuration-properties-table.adoc:513]: Hi Reshma, Here it is : https://github.com/killbill/killbill/pull/1936

File: userguide/getting-started/includes/tomcat.adoc [hunk 6]
```diff
@@ -20,7 +20,7 @@ The first step in installing Kill Bill in Tomcat is to download and install Tomc
  
 
 
-. Ensure that you have *JDK 11* installed (It can be downloaded from https://www.oracle.com/in/java/technologies/javase/jdk11-archive-downloads.html[here])
+. Ensure that you have *JDK 11* installed ( https://www.openlogic.com/openjdk-downloads[OpenJDK] is the default requirement, though Kill Bill supports all JDKs. Oracle JDK can be downloaded from https://www.oracle.com/in/java/technologies/javase/jdk11-archive-downloads.html[here])
```
Comment (pierre) [inline userguide/getting-started/includes/tomcat.adoc]: https://openjdk.org/ is the official OpenJDK website

Comment (reshmabidikar) [reply userguide/getting-started/includes/tomcat.adoc]: Could you please also include some text to say that all JVMs are supported as mentioned by Stephane on slack?

**Review by reshmabidikar — APPROVED** [review]

**Pull request merged**